pywemo/ouimeaux_device/bridge.py

```diff
@@ -105,7 +105,7 @@ def _update_state(self, status):
             if not value:
                 value = None
             elif ':' in value:
-                value = tuple(int(v) for v in value.split(':'))
+                value = tuple(int(float(v)) for v in value.split(':'))
             else:
                 value = int(value)
```

Comment:
Could you also do `int(round(float(v))) `. Probably not needed for your devices - but could help with some other device!